core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -39,6 +39,12 @@ import kafka.server.KafkaServer
 /* We have some tests in this class instead of `BaseConsumerTest` in order to keep the build time under control. */
 class PlaintextConsumerTest extends BaseConsumerTest {
 
+  @Before
+  override def setUp() {
+    super.setUp
+    this.consumerConfig.remove(ConsumerConfig.CLIENT_ID_CONFIG)
```

Comment:
Do we really need this, as long as it can be overridden in each specific test case?